libs/events/src/types/orders.ts

```diff
@@ -12,31 +17,22 @@ export type BaseOrderPayload = {
   inputToken: TokenInfo
   outputToken: TokenInfo
   receiver?: string
-}
-
-export type OnPostedOrderPayload = BaseOrderPayload & {
   orderCreationHash?: string
   isEthFlow?: boolean
 }
 
-export type OnFulfilledOrderPayload = {
-  chainId: SupportedChainId
-  order: EnrichedOrder
+export type OnFulfilledOrderPayload = BaseOrderPayload & {
+  //
```

Comment:
Why the empty obj? Does the linter forbid a simple assignment?

Reply:
This was just a placeholder :) 
Removed it